### PR TITLE
Add Crystallize as a headless CMS

### PR DESCRIPTION
**Pull request by danisal**
- Add Crystallize markdown
- Add photos

### CODE REVIEW

File: content/projects/crystallize.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Crystallize
+homepage: https://www.crystallize.com/
+twitter: CrystallizeAPI
+repo: https://github.com/CrystallizeAPI
+opensource: "No"
+typeofcms: "API Driven"
+supportedgenerators:
+    - All
+description: Crystallize is a GraphQL PIM service with a twist; manage structured product information and rich marketing content. A fast GraphQL API for building juicy headless customer experiences in any channel. Ideal for JAMStack & React. Sign up to your [free tier Crystallize account](https://crystallize.com/signup) and start building.
```
Comment (huguestennier) [inline content/projects/crystallize.md:10]:
```suggestion
description: Crystallize is a GraphQL PIM service with a twist; manage structured product information and rich marketing content. A fast GraphQL API for building juicy headless customer experiences in any channel.
```

Can you make the description a bit shorter so it fits better with the other cards around it? Added a suggestion to remove the link as the card doesn’t support markdown at the moment

File: content/projects/crystallize.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+<img class="simple" src="/images/crystallize-fast-graphql-api.png" alt="GraphQL API on Crystallize" />
+
+### Scalable cloud
+
+Your content is stored on our servers, they will be safe, secure and delivered super fast. Images are transcoded and converted to fast WebP formats. HD video is transcoded and ready for streaming direct from our Edge network.
+
+### Structured Content
+
+Model content your way using [shapes](https://crystallize.com/learn/concepts/pim/shape) in Crystallize. Build a [custom content model](https://crystallize.com/blog/structured-content-pim-withhttps://crystallize.com/learn/best-practices/information-architecture/content-modelling) instead of dealing with fixed, predefined templates.
```
Comment (huguestennier) [inline content/projects/crystallize.md:52]: `https://crystallize.com/blog/structured-content-pim-withhttps://crystallize.com/learn/best-practices/information-architecture/content-modelling` link doesn’t work

File: content/projects/crystallize.md (hunk 1 above)
Comment (huguestennier) [reply content/projects/crystallize.md:10]: Screenshot for reference: 

Screen Shot 2020-07-06 at 2 17 15 PM

Comment (danisal) [reply content/projects/crystallize.md]: Made the changes

File: content/projects/crystallize.md (hunk 2 above)
Comment (danisal) [reply content/projects/crystallize.md]: Update the link

**Comment by zachleat** [comment]
Thanks!

**Pull request merged**